h2o-assemblies/main/build.gradle

```diff
@@ -83,6 +83,8 @@ shadowJar {
   exclude 'test.properties'
   exclude 'cockpitlite.properties'
   exclude 'devpay_products.properties'
+  exclude 'org/apache/logging/log4j/core/lookup/JndiLookup.class'
+  exclude 'org/apache/logging/log4j/core/lookup/JmxRuntimeInputArgumentsLookup.class'
```

Comment:
do you need to remove 2nd class?

Reply:
after discussion with @coolsat there is a concern that the change could impact "innocent" customers (who needs JNDI)

Reply:
ok, reverting back to just version upgrade